CHANGES_UNRELEASED.md

```diff
@@ -19,7 +19,26 @@ Use the template below to make assigning a version number during the release cut
 
 -->
 
+## Android
+
+### ⚠️ Breaking Changes ⚠️
+  - Many error classes have been renamed from `FooError` or `FooErrorException` to just `FooException`,
+    to be more in keeping with Java/Kotlin idioms.
+
+## Logins
+
+### ⚠️ Breaking Changes ⚠️
+  - Members of the `LoginsStoreError` enum no longer have a `message` property giving a detailed error message.
```

Comment:
is this still true with https://github.com/mozilla/uniffi-rs/pull/1025/files? since it's a flat enum?

If it's still true, nit:
```suggestion
  - Members of the `LoginsStorageError` enum no longer have a `message` property giving a detailed error message.
```

Reply:
No this is not true anymore and will update this -- nice catch!